org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/TargetPlatformProject.xtend

```diff
@@ -77,6 +77,7 @@ class TargetPlatformProject extends ProjectDescriptor {
 					�ENDIF�
 				</location>
 				<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+					<unit id="io.github.classgraph" version="4.8.117.v20210930-0420"/>
```

Comment:
Does it make sense to go straight to 137? https://github.com/classgraph/classgraph/releases/tag/classgraph-4.8.137

Reply:
Maybe I misunderstood the purpose of this merge request.

LGTM

Reply:
no this can be done once we have something to put to umbrella only (next step)